traja/models/train.py

```diff
@@ -11,7 +11,7 @@ class HybridTrainer(object):
     """
     Wrapper for training and testing the LSTM model
     Args:
-        model_type: Type of model should be "LSTM"
+        model_type: Type of model should be 'LSTM'
```

Comment:
`model_type` is injected through the model itself and is not needed here. Also, the corresponding __init__ variable doesn't exist.

Reply:
This has always been there in `train.py`
All I did was change `"` -> `'` for consistency.

Reply:
Please remove the argument then, since it isn't being used

Reply:
Done.